central_service/app/frontend/app/templates/components/guides/create-lambda-instance.hbs

```diff
@@ -0,0 +1,163 @@
+<h3>Lambda Instance Creation</h3>
+
+<p>This guide assumes you already have a LoD service VM up and running. If not, read {{#link-to 'index'}}this{{/link-to}}. You can create a Lambda Instance through the control panel of the service VM. The images below will guide you step by step in the creation of your first Lambda Instance.</p>
+
+<div class="row margin-bottom">
+	<div class="col-sm-6">
+		<div class="row margin-bottom">
+			<div class="col-sm-3">
+				<a href="#" data-toggle="modal" data-target="#LoginPage">
+					<img class="img-responsive" src="assets/img/faqs/login-sm.png" alt="Photo" >
+				</a>
+				<p class="text-center">Step 1</p>
+			</div>
+			<div class="col-sm-1 text-center">
+				<span class="fa fa-arrow-right"></span>
+			</div>
+			<div class="col-sm-3">
+				<a href="#" data-toggle="modal" data-target="#createStep2">
+					<img class="img-responsive" src="assets/img/faqs/lambda-instance/create-sm.png" alt="Photo">
+				</a>
+				<p class="text-center">Step 2</p>
+			</div>
+			<div class="col-sm-1 text-center">
+				<span class="fa fa-arrow-right"></span>
+			</div>
+			<div class="col-sm-3">
+				<a href="#" data-toggle="modal" data-target="#createStep3">
+					<img class="img-responsive" src="assets/img/faqs/lambda-instance/create/step3-sm.png" alt="Photo">
+				</a>
+				<p class="text-center">Step 3</p>
+			</div>
+		</div>
+		<div class="row ">
+			<div class="col-sm-12 col-md-offset-9">
+				<span class="fa fa-arrow-down"></span>
+			</div>
+		</div>
+		<div class="row ">
+			<div class="col-sm-3 col-md-offset-4">
+				<a href="#" data-toggle="modal" data-target="#createStep5">
+					<img class="img-responsive" src="assets/img/faqs/lambda-instance/create/step5-sm.png" alt="Photo" >
+				</a>
+				<p class="text-center">Step 5</p>
+			</div>
+			<div class="col-sm-1 text-center">
+				<span class="fa fa-arrow-left"></span>
+			</div>
+			<div class="col-sm-3">
+				<a href="#" data-toggle="modal" data-target="#createStep4">
+					<img class="img-responsive" src="assets/img/faqs/lambda-instance/create/step4-sm.png" alt="Photo" >
+				</a>
+				<p class="text-center">Step 4</p>
+			</div>
+		</div>
+	</div>
+</div><!-- /.col -->
+
+<div class="row">
+	<div class="col-md-12">
+		<ol>
+			<li>Login the service dashboard using your ~okeanos token(you can find your ~okeanos token <a href="https://accounts.okeanos.grnet.gr/ui/api_access">here</a>)</li>
```

Comment:
Please replace 'login the' with 'log into', as in other guides. Also, insert a space between token and '('